backend/api.test/Database/Models.cs

```diff
@@ -71,62 +71,5 @@ public void TestNegativaRotation()
                 3.0
             );
         }
-
-        [Fact]
-        public void AssertCoordinateConversion()
```

Comment:
Have these tests been moved somewhere or just deleted? Sorry it was hard to tell

Reply:
This tests only tests function that are in the test folder and not used in the rest of the application, this should probably have been a separate commit indeed explaining why they are deleted